mono/metadata/w32handle.c

```diff
@@ -282,36 +280,6 @@ mono_w32handle_init (void)
 	initialized = TRUE;
 }
 
-void
-mono_w32handle_cleanup (void)
```

Comment:
This change doesn't belong to a PR named "Unify WaitHandle.Wait{One,Any,All} icalls"